homeassistant/components/go2rtc/__init__.py

```diff
@@ -224,7 +224,12 @@ async def async_handle_async_webrtc_offer(
         ):
             await self._rest_client.streams.add(
                 camera.entity_id,
-                [stream_source, f"ffmpeg:{camera.entity_id}#audio=opus"],
+                [
+                    stream_source,
+                    # As ffmpeg is the second stream and it's refereing to the first
+                    # one, we can lower the log level from warning to debug
```

Comment:
There's a typo in refereing, I also find the comment a bit hard to understand. Can you try to rephrase?